cirq-web/cirq_ts/e2e/bloch_sphere/bloch_sphere_e2e_test.ts

```diff
@@ -0,0 +1,140 @@
+// Copyright 2021 The Cirq Developers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import puppeteer from 'puppeteer';
+import {expect} from 'chai';
+import {readFileSync} from 'fs';
+import pixelmatch from 'pixelmatch';
+import * as PNG from 'pngjs';
+import * as temp from 'temp';
+import * as path from 'path';
+
+/**
+ * Generates an HTML script with the current repository bundle
+ * that we will use to compare.
+ */
+
+// Due to the path, reading the file will only work by running this file in the same directory
+// as the package.json file.
+const bundle_string = readFileSync('dist/bloch_sphere.bundle.js');
+const browserContent = `
+<!doctype html>
+<meta charset="UTF-8">
+<html lang="en">
+    <head>
+    <title>Cirq Web Development page</title>
+    </head>
+    <body>
+    <div id="container"></div>
+    <script>${bundle_string}</script>
+    <script>
+        renderBlochSphere('{"radius": 5}', 'container').addVector();
+    </script>
+    </body>
+</html>
+`;
+
+const newVectorBrowserContent = `
+<!doctype html>
+<meta charset="UTF-8">
+<html lang="en">
+    <head>
+    <title>Cirq Web Development page</title>
+    </head>
+    <body>
+    <div id="container"></div>
+    <script>${bundle_string}</script>
+    <script>
+        renderBlochSphere('{"radius": 5}', 'container').addVector('{"x": 1,"y": 1, "z": 2, "length": 5}');
+    </script>
+    </body>
+</html>
+`;
+
+/**
+ * Testing to see if they look the same.
+ */
+
+// Automatically track and cleanup files on exit
+temp.track();
+
+describe('Check Bloch Sphere looks correct', () => {
```

Comment:
nit: refactor the `describe` / `it` sections to describe(Bloch sphere), it(matches the default state)